UCR/rdf-star-ucr.html

```diff
@@ -287,6 +287,76 @@ <h4>annotate commit deltas</h4>
           so that a triple can be searched for across commit history in SPARQL*.
         </p>
       </section>
+
+      <section id="rdfsSupport">
+        <h4>SPARQL*/RDF* and RDFS support</h4>
```

Comment:
```suggestion
        <h4>SPARQL-star, RDF-star, and RDFS support</h4>
```

Reply:
This is as written by the contributor.

Reply:
"This is as written by the contributor." Sure. It should also be changed to match group decisions, in the process of adding it to group documents. I can think of no reason why we must add (and merge) the UC as submitted before making such editorial changes as this, rather than making these changes during the add/merge process.